meshroom/ui/qml/GraphEditor/Edge.qml

```diff
@@ -41,6 +41,8 @@ Shape {
         startY: root.startY
         fillColor: "transparent"
         strokeColor: "#3E3E3E"
+        strokeStyle: edge != undefined && ((edge.src != undefined && edge.src.isOutput) || edge.dst == undefined) ? ShapePath.SolidLine : ShapePath.DashLine
+        dashPattern: [4, 4]
```

Comment:
A dash pattern of [6,4] could more adapted to get a stronger feeling of a dashed line (VS a dotted line).
Also, avoid variation in dash pattern when the stroke is hovered by compensating strokeWidth:

```
ShapePath {
  ...
  // final visual width of this path (never below 1)
  readonly property real visualWidth: Math.max(strokeWidth, 1)
  ...
  dashPattern: [6/visualWidth, 4/visualWidth]
}
```